src/main/scala/algolia/responses/Logs.scala

```diff
@@ -39,7 +39,15 @@ case class Log(
     processing_time_ms: String,
     index: Option[String],
     query_nb_hits: Option[String],
-    exhaustive: Option[Boolean]
+    exhaustive: Option[Boolean],
+    inner_queries: Seq[InnerQuery]
```

Comment:
Not sure about this language, but this is also optional: `Option[Seq[InnerQuery]]`.